source/initial_temperature/adiabatic.cc

```diff
@@ -289,6 +344,15 @@ namespace aspect
                              "This function is one-dimensional and depends only on depth. The format of this "
                              "functions follows the syntax understood by the "
                              "muparser library, see Section~\\ref{sec:muparser-format}.");
+          prm.declare_entry ("Use ASCII file for seafloor age", "false",
+                             Patterns::Bool (),
+                             "Whether to define seafloor ages with an ASCII data file.");
```

Comment:
```suggestion
                             "Whether to define seafloor ages with an ASCII data file. "
                              "Note that if the top boundary is not included in the list "
                              "of fixed temperature boundary indicators, the values"
                              "from the ascii file will be ignored and the initial temperature "
                              "profile will be entirely derived from the adiabatic profile.");
```

Reply:
@danieldouglas92 @jdannberg - How does this sound? Alternatively, we could put in an Assert that in order to use the ascii data option one must have the top boundary in the list of fixed temperature boundary indicators?

Reply:
I think this could go in the general description of the plugin. After all, the "Age top boundary layer" parameter will also be ignored if the temperature is not fixed at the top boundary, so this is not specific to reading in the ascii data file.

Reply:
I've addressed all the other comments, besides the issue with the error loading in the ASCII file. What do you think is the best course of action regarding this?